charts/cu-cp/templates/deployment-ds-ctrl.yaml

```diff
@@ -0,0 +1,141 @@
+{{- include
+      "accelleran.common.deployment"
+      (fromYaml (include "accelleran.cu-cp.ds-ctrl.deployment.args" $))
+-}}
```

Comment:
We still do need that ID, but we just inject the pod name now (I made this so it gets accepted, before it needed to be with alphanumeric characters and an underscore).

DS controller is a bit more unique though with it's multi container pod. There we need to append the "container index" to the pod name, which we do through the init container (it can inject that in the environment of the executable through a file `/config/env`).